index.html

```diff
@@ -253,10 +253,11 @@ <h2>So, what's the verdict?</h2>
 <p>The ActivityPub standard briefly mentions authentication (and authorization), but omits specific needs or use cases for them. Over time, two clear uses for authentication in the fediverse have emerged.</p>
 <p>The first is proving and verifying that a given user created a given piece of data or performed a given action. This is the classic attribution problem in identity-based networks. (Note that this is separate from authorization or access control, ie determining whether a given user is <em>allowed</em> to access a given piece of data or perform a given action.)</p>
 <p>At a baseline level, this works. HTTP Signatures attached to ActivityPub inbox delivery requests can effectively verify the actor - or at least the server - who sent them. However, these signatures are ephemeral. They only authenticate HTTP requests, not long-lived data. We'd often like to authenticate objects and activities outside of inbox delivery requests, eg during <a href="https://www.w3.org/TR/activitypub/#inbox-forwarding">inbox forwarding</a>, or after they were initially created. HTTP Signatures can't do this.</p>
-<p>Some fediverse projects like Mastodon use <a href="https://web.archive.org/web/20170923124140/https://w3c-dvcg.github.io/ld-signatures/">LD Signatures 1.0</a> for those purposes instead, which works, but isn't widely supported. Even Mastodon's support itself is limited and discouraged. <a href="https://docs.joinmastodon.org/spec/security/#ld">From their docs</a>:</p>
+<p>Some fediverse projects based on the Mastodon API use <a href="https://web.archive.org/web/20170923124140/https://w3c-dvcg.github.io/ld-signatures/">LD Signatures 1.0</a> for object-authentication purposes instead, which works, but is unevenly supported within the Mastodon-API ecosystem. Even Mastodon's ongoing support for it is limited and deprecated, as, like Cavage-12, the final normative recommendation is not backwards-compatible. <a href="https://docs.joinmastodon.org/spec/security/#ld">From Mastodon's docs</a>:</p>
```

Comment:
I don't quite follow this. LD Sigs are unrelated to the Mastodon API, right? And other non-Mastodon servers emit (and maybe consume?) them too.

I'm totally with you that object-level/self-signed/embedded auth is great, and a meaningful improvement over HTTP Sigs! I tried to wordsmith this paragraph carefully to only discourage LD Sigs specifically, not object-level auth in general. Did it not come across that way?

Reply:
Perhaps a point of clarification: not Mastodon's API (which is proprietary) but Mastodon's AP implementation.